assignment/closure/closure2.js

```diff
@@ -0,0 +1,15 @@
+var saveForLater=function()
+{
+    return function()
+    {
+        return "MyName";
```

Comment:
This should not be hard coded. There is no closure here aslan :) He is passing you the string that you need to return in the constructor.